packages/deployments/contracts/contracts/messaging/RootManager.sol

```diff
@@ -235,6 +507,7 @@ contract RootManager is ProposedOwnable, IRootManager, WatcherClient, DomainInde
    * @param _inbound The inbound root coming from the given domain.
    */
   function aggregate(uint32 _domain, bytes32 _inbound) external whenNotPaused onlyConnector(_domain) {
+    if (optimisticMode) revert RootManager_OptimisticModeOn();
```

Comment:
@preethamr we should make sure that on the off-chain side we are halting the `processFromRoot` tasks appropriately

pretty sure we already talked about that though